t/plugin/proxy-rewrite.t

```diff
@@ -1311,3 +1311,56 @@ host: test.com:6443
 x-real-ip: 127.0.0.1
 --- no_error_log
 [error]
+
+
+
+=== TEST 45: set route(rewrite host)
```

Comment:
The title is incorrect. We don't rewrite the host

Reply:
Done